Marlin/Configuration_adv.h

```diff
@@ -135,7 +135,7 @@
 #endif
 
 // Show extra position information in M114
-//#define M114_DETAIL
+#define M114_DETAIL
```

Comment:
We definitely don't want to include changes to the base `Configuration_adv.h` in a PR.